jvm/src/vyxal/gen/GenerateDocs.scala

```diff
@@ -5,15 +5,17 @@ import vyxal.parsing.Lexer
 
 /** For generating elements.txt and trigraphs.txt. See build.sc */
 private object GenerateDocs:
-  def generate(): (String, String) = (elements(), trigraphs())
+  def generate(): (String, String, String) =
+    (elements(), trigraphs(), elementTable())
 
   def elements(): String =
     val sb = StringBuilder()
     Elements.elements.values.toSeq
       .sortBy { elem =>
         // Have to use tuple in case of digraphs
         (
-          Lexer.Codepage.indexOf(elem.symbol.charAt(0)),
+          Lexer.Codepage.indexOf(elem.symbol.charAt(0)) +
+            (if "#∆øÞ".contains(elem.symbol.charAt(0)) then 200 else 0),
```

Comment:
Is it safe to hardcode 200 and 400?

Reply:
Yeah. It's just a value to make the value of digraphs higher than other elements. That makes them display at the bottom of the table/elements.txt file. That 200 should be 400 though because hash happens to be less than 256 when added with 200